components/molecules/ContributorProfileHeader/contributor-profile-header.tsx

```diff
@@ -182,69 +179,70 @@ const ContributorProfileHeader = ({
                 </Button>
 
                 {!isOwner && (
-                  <DropdownMenuTrigger title="More options" className="p-2 mr-3 bg-white rounded-full cursor-pointer ">
+                  <PopoverTrigger title="More options" className="p-2 mr-3 bg-white rounded-full cursor-pointer ">
                     <TfiMoreAlt size={20} className="" />
-                  </DropdownMenuTrigger>
+                  </PopoverTrigger>
                 )}
               </div>
 
-              <DropdownMenuContent align="end" className="flex flex-col gap-1 py-2 rounded-lg">
+              <PopoverContent
+                align="end"
+                className="flex flex-col gap-1 py-2 px-2 !z-40 w-36 text-sm font-medium rounded-lg bg-white outline-none"
+              >
                 {user ? (
                   !isOwner && (
                     <>
-                      <DropdownMenuItem
+                      <button
                         onClick={handleFollowClick}
-                        className="rounded-md flex items-center gap-1 !cursor-pointer [&>span>span:nth-child(1)]:hover:hidden [&>span>span:nth-child(1)]:focus:hidden [&>span>span:nth-child(2)]:hover:inline [&>span>span:nth-child(2)]:focus:inline"
+                        className="rounded-md flex gap-1 !cursor-pointer hover:text-sauced-orange hover:bg-orange-100 [&>span>span:nth-child(1)]:hover:hidden [&>span>span:nth-child(1)]:focus:hidden [&>span>span:nth-child(2)]:hover:inline [&>span>span:nth-child(2)]:focus:inline"
```

Comment:
having this odd behavior when i try to hover by the end right character 
![Screenshot 2023-09-19 at 18 39 00](https://github.com/open-sauced/app/assets/62995161/c2e57d1c-e917-43ef-a9b9-ad21efe5ca98)
```suggestion
                        className="rounded-md flex gap-1 !cursor-pointer hover:text-sauced-orange hover:bg-orange-100"
```
Those pseudo-classes are not needed anymore.